app/models/csv_upload_job.rb

```diff
@@ -0,0 +1,34 @@
+class CSVUploadJob
+  attr_reader :batch
+
+  def initialize(batch)
+    @batch = batch
+  end
+
+  def payload
+    output_documents = batch.appointment_summaries.map do |appointment_summary|
+      OutputDocument.new(appointment_summary)
+    end
+
+    CSVRenderer.new(output_documents).render
+  end
+
+  def payload_path
+    upload_path('csv')
+  end
+
+  def trigger
+    ''
+  end
+
+  def trigger_path
+    upload_path('trg')
```

Comment:
That directory exists as part of their "standard" SFTP setup. For this integration they're expecting trigger files alongside the data files.